xml/containers-docker-installation.xml

```diff
@@ -89,7 +89,7 @@
     </para>
 <screen>&prompt.sudo;systemctl enable docker.service</screen>
     <para>
-     This automatically enables <systemitem>docker.socket</systemitem> in
+     This automatically enables <systemitem>docker.socket</systemitem> as a
```

Comment:
How about 'This also automatically enables <systemitem>docker.socket</systemitem>.'

Reply:
+1 for removing `as a consequence`.